target/linux/ramips/dts/mt7621_ubnt_edgerouter-x-sfp.dts

```diff
@@ -5,13 +5,71 @@
 / {
 	model = "Ubiquiti EdgeRouter X SFP";
 	compatible = "ubnt,edgerouter-x-sfp", "mediatek,mt7621-soc";
+
+	sfp_eth5: sfp_eth5 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c>;
+		mod-def0-gpio = <&expander0 5 GPIO_ACTIVE_LOW>;
+		maximum-power-milliwatt = <1000>;
+	};
 };
 
 &i2c {
 	status = "okay";
 
-	pca9555@25 {
+	/*
+	 * PCA9655 GPIO expander
+	 *  0-POE power port eth0
+	 *  1-POE power port eth1
+	 *  2-POE power port eth2
+	 *  3-POE power port eth3
+	 *  4-POE power port eth4
+	 *  5-SFP_MOD_DEF0#
+	 *  6-
+	 *  7-
+	 *  8-Pull up to VCC
+	 *  9-Pull down to GND
+	 * 10-Pull down to GND
+	 * 11-Pull down to GND
+	 * 12-Pull down to GND
+	 * 13-Pull down to GND
+	 * 14-Pull down to GND
+	 * 15-Pull down to GND
+	 */
+	expander0: pca9555@25 {
 		compatible = "nxp,pca9555";
+		interrupt-parent = <&gpio>;
+		interrupts = <8 IRQ_TYPE_EDGE_FALLING>;
+		gpio-controller;
+		#gpio-cells = <2>;
 		reg = <0x25>;
 	};
 };
+
+&gpio {
+	sfp_i2c_clk_gate {
+		gpio-hog;
+		gpios = <7 GPIO_ACTIVE_LOW>;
+		output-high;
+	};
+};
+
+&mdio {
+	ephy5: ethernet-phy@7 {
```

Comment:
That should be labelled according to the reg property, i.e. phy7 (or ephy7).

Reply:
Fixed.